lib/compress.js

```diff
@@ -774,13 +767,6 @@ merge(Compressor.prototype, {
                 }
             }
 
-            // Remove extraneous empty statments in block after removing var definitions.
-            // Leave at least one statement in `statements`.
```

Comment:
There was put in to work around a bug of some sort. Hopefully I added a test for it.